feature-toggle/src/app/demo/demo.component.html

```diff
@@ -0,0 +1,12 @@
+<p>demo works!</p>
```

Comment:
remove

Reply:
Whole component remove, it was for testing